tox.ini

```diff
@@ -20,26 +20,26 @@ passenv =
     *
 usedevelop = false
 deps =
-    pytest==3.10.0
-    pytest-travis-fold==1.3.0
-    pytest-cov==2.6.0
-    PyYAML==5.1.2
+    pytest~=7.0.1
+    pytest-travis-fold~=1.3.0
+    pytest-cov~=3.0.0
+    PyYAML~=6.0
 commands =
     {posargs:pytest -m "not toml_installed" --cov --cov-report=term-missing -vv tests}
 
-[testenv:{py36,py37,py38,py39}-toml]
+[testenv:{py37,py38,py39,py310}-toml]
 deps =
     {[testenv]deps}
-    toml==0.10.2
+    toml~=0.10.2
 commands =
     {posargs:pytest -m "not toml_not_installed" --cov --cov-report=term-missing -vv tests}
 
 
 [testenv:check]
 deps =
-    black==20.8b1
-    flake8==3.7.8
-    mypy==0.730
+    black~=22.2.1
```

Comment:
This seems to be [causing problems for Travis](https://app.travis-ci.com/github/seddonym/import-linter/jobs/560928839), maybe we should downgrade?

Reply:
That's probably because the latest `black` version is `22.1.0` - `22.2.1` is not released yet

Reference: https://pypi.org/project/black/#history

Reply:
That's was a typo, fixed now